### PR TITLE
Update production 071216

### PR DESCRIPTION
**Pull request by akainic**

### CODE REVIEW

File: test/data/conformance/no-financial.xml [hunk 1]
```diff
@@ -1 +1,5 @@
...
.../eeSummary:powIndicator></eeSummary:prisonerOfWarInfo><eeSummary:purpleHeart xmlns:eeSummary="http://jaxws.webservices.esr.med.va.gov/schemas"><eeSummary:indicator>false</eeSummary:indicator></eeSummary:purpleHeart><eeSummary:personInfo xmlns:eeSummary="http://jaxws.webservices.esr.med.va.gov/schemas"><eeSummary:firstName>FirstName</eeSummary:firstName><eeSummary:middleName>MiddleName</eeSummary:middleName><eeSummary:lastName>ZZTEST</eeSummary:lastName><eeSummary:suffix>Jr.</eeSummary:suffix><eeSummary:ssnText>111111234</eeSummary:ssnText><eeSummary:gender>F</eeSummary:gender><eeSummary:dob>01/02/1923</eeSummary:dob><eeSummary:mothersMaidenName>Maiden</eeSummary:mothersMaidenName><eeSummary:placeOfBirthCity>Springfield</eeSummary:placeOfBirthCity><eeSummary:placeOfBirthState>AK</eeSummary:placeOfBirthState></eeSummary:personInfo></va:summary><va:applications><va:applicationInfo><va:appDate>2015-10-21</va:appDate><va:appMethod>1</va:appMethod></va:applicationInfo></va:applications></va:form><va:identity xmlns:va="http://va.gov/schema/esr/voa/v1"><va:authenticationLevel><va:type>100</va:type><va:value>anonymous</va:value></va:authenticationLevel></va:identity></ns1:submitFormRequest>
```
Comment (markolson) [inline test/data/conformance/no-financial.xml]: merge conflict

Comment (akainic) [inline test/data/conformance/no-financial.xml]: Ah yes, fixed!

**Comment by markolson** [comment]
LGTM if tests pass

**Comment by markolson** [comment]
Added a fix for the broken tests - trailing newlines got into the xml files which was causing a strict comparison to fail.

**Comment by markolson** [comment]
@akainic this should be good to go now

**Pull request merged**